apps/core/scenes/results/Results.js

```diff
@@ -52,9 +52,16 @@ type Props = {|
   +bags: number,
   +sortBy: string,
   +layout: number,
+  +router: any,
```

Comment:
Can't we use here something more specific?

Reply:
Already changed it for the `router.query` (which is an object, not very specific either)